sources/packages/api/src/esdc-integration/sin-validation/sin-validation-files/sin-validation-file-header.ts

```diff
@@ -0,0 +1,59 @@
+import { FixedFormatFileLine } from "../../../services/ssh/sftp-integration-base.models";
+import { RecordTypeCodes } from "../models/sin-validation-models";
+import {
+  DATE_FORMAT,
+  NUMBER_FILLER,
+  SPACE_FILLER,
+} from "../../../esdc-integration/models/esdc-integration.model";
+import { getDateOnlyFromFormat, StringBuilder } from "../../..//utilities";
+
+/**
+ * Header of an ESDC SIN validation file.
+ * Shared between the file send and received.
+ * The documentation about it is available on the document 'SIN Check File Layouts 2019.docx'
+ */
+export class SINValidationFileHeader implements FixedFormatFileLine {
+  /**
+   * File record type.
+   */
+  recordTypeCode: RecordTypeCodes;
+  /**
+   * File sequential number.
+   */
+  batchNumber: number;
+  /**
+   * File creation time.
+   */
+  processDate: Date;
+
+  /**
+   * Get the information as a fixed line format to be
+   * added to the file uploaded to the SFTP.
+   * @returns fixed line formatted.
+   */
+  getFixedFormat(): string {
+    const record = new StringBuilder();
+    record.append(this.recordTypeCode);
+    record.appendWithStartFiller(this.batchNumber, 8, NUMBER_FILLER);
+    record.appendDate(this.processDate, DATE_FORMAT);
+    record.append("BC");
```

Comment:
now we have the Province code in our DB right? can we use it from there? instead of hard coding.

Reply:
These are fixed province codes. I will create a const only.

Reply:
Constant add for ESDC SIN validation scope only.